lib/common/database.dart

```diff
@@ -226,7 +226,23 @@ class DatabaseService {
     }
     return Task();
   }
+  Future<List<Task>> getTasksOfName(String taskName) async {
```

Comment:
Can we add a newline here? A lot of files are not up to standard with the new coding standards, maybe we can fix them all at once then instead?